openpathsampling/movechange.py

```diff
@@ -46,6 +47,12 @@ def __init__(self, subchanges=None, samples=None, mover=None, details=None):
             self.samples = []
         else:
             self.samples = samples
+
+        if input_samples is None:
+            self.input_samples = list()
```

Comment:
Just a note because I read this recently. For some reason `[]` is faster than `dict()`. I would have guessed that
the interpreter treats it the same, but in theory you could alter the behaviour of dict and hence you need to do a function call while `[]` is really syntactic and will always give an empty list. My test showed a factor of 2x.